docs/api/retrieval_model_clients/AzureCognitiveSearch.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 2
+sidebar_position: 3
```

Comment:
Place DatabricksRM second in the list of retrievers documented at https://dspy-docs.vercel.app/api/category/retrieval-model-clients